specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/blobAuditing.json

```diff
@@ -124,6 +124,51 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/auditingSettings": {
```

Comment:
Each resource type should have a collection/list API and a point/get-by-name API. Is this a singleton where the collection would return a single item? Or are there numerous items that can be returned?  Using a singleton as example, we should have both:
'/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/auditingSettings'
and
'/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/auditingSettings]/default'
paths

Reply:
This is a singleton, a collection of one item is returned.
The `GET` API is described by `"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/auditingSettings/{blobAuditingPolicyName}"`, while the `LIST` API is described by `"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/auditingSettings"`